client/extensions/woocommerce/state/ui/orders/reducer.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { combineReducers, keyedReducer } from 'state/utils';
+import { WOOCOMMERCE_UI_ORDERS_SET_PAGE } from 'woocommerce/state/action-types';
+
+/**
+ * Tracks the current page of orders displayed for the current site.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export function currentPage( state = 0, action ) {
```

Comment:
I was having trouble with keyedReducer deleting the state when it matched the initial state, but that's [intended behavior](https://github.com/Automattic/wp-calypso/blob/master/client/state/utils.js#L119). I did this as a hack around that issue so my tests would run, but really I should have updated my tests — which I've done in my latest commit.